includes/templates/responsive_classic/css/stylesheet.css

```diff
@@ -20,15 +20,15 @@ a:active {}
 h1 {font-size:1.8em;line-height:25px;}
 h2 {font-size:1.4em;}
 h3 {font-size:1.3em;}
-h4, h5, h6, h4.optionName, LEGEND, ADDRESS, .larger{font-size:1.1em;}
+h4, h5, h6, LEGEND, ADDRESS, .larger{font-size:1.1em;}
 h1, h2, h3, h4, h5, h6 {margin:20px 0;}
 .biggerText {font-size:1.2em;line-height:160%;}
 P, ADDRESS {padding:0.5em;}
 ADDRESS {font-style:normal;}
-HR {height: 1px;margin-top:0.5em;border:none;}
+HR {height: 1px;margin-top:0.5em;}
 #logo, .centerBoxContents, .specialsListBoxContents, .categoryListBoxContents, .centerBoxContentsAlsoPurch, .attribImg {float:left;}
-.centeredContent, TH, #cartEmptyText, #cartBoxGVButton, #cartBoxEmpty, #cartBoxVoucherBalance, #navCatTabsWrapper, #navEZPageNextPrev, #bannerOne, #bannerTwo,  #bannerThree,  #bannerFour,  #bannerFive,  #bannerSix, #siteinfoLegal, #siteinfoCredits, #siteinfoStatus, #siteinfoIP, .center, .cartRemoveItemDisplay, .cartQuantityUpdate, .cartQuantity, .cartTotalsDisplay, #cartBoxGVBalance, .leftBoxHeading, .centerBoxHeading,.rightBoxHeading, .productListing-data, .accountQuantityDisplay, .ratingRow, LABEL#textAreaReviews, #productMainImage, #reviewsInfoDefaultProductImage, #productReviewsDefaultProductImage, #reviewWriteMainImage, .centerBoxContents, .specialsListBoxContents, .categoryListBoxContents, .additionalImages, .centerBoxContentsSpecials, .centerBoxContentsFeaturedCategories, .centerBoxContentsAlsoPurch, .centerBoxContentsFeatured, .centerBoxContentsNew, .gvBal, .attribImg {text-align:center;}
```

Comment:
`.accountQuantityDisplay` is not assigned to `text-align: center` anywhere else.

Reply:
Oops. Yes it is.